test/prettier.js

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env node
+/* eslint-disable node/shebang */
```

Comment:
Why is this file executable? `RuleTester` uses mocha internally, so it seems like this would throw an error if it was just run with `node`.